examples-next/ecommerce/mutations.ts

```diff
@@ -42,34 +49,40 @@ export const extendGraphqlSchema = graphQLSchemaExtension({
 
         // 3. Create the Payment Intent, given the Payment Method ID
         // by passing confirm: true, We do stripe.paymentIntent.create() and stripe.paymentIntent.confirm() in 1 go.
-        // FIXME: How do we test this? Is this going to charge someone's card?
-        const charge = { id: 'MADE UP', amount, token };
-        // const charge = await stripe.paymentIntents.create({
-        //   amount,
-        //   currency: 'USD',
-        //   confirm: true,
-        //   payment_method: token,
-        // });
+        // You can bypass stripe by using this:
+        // const charge = { id: `fake-charge-${Date.now()}`, amount, token };
+        const charge = await stripe.paymentIntents.create({
+          amount,
+          currency: 'USD',
+          confirm: true,
+          payment_method: token,
+        }).catch(err => {
+          console.log('shoot!');
+          console.log(err);
+        }) as any; // TODO: Stripe Type?
+        console.log(`Back from stripe!`, charge.id);
 
         // 4. Convert the CartItems to OrderItems
-        const orderItems = User.cart.map((cartItem: any) => {
+        // TODO Type CartItem and OrderItem. Can they be generated?
+        const orderItems = User.cart.map((cartItem) => {
           const orderItem = {
             name: cartItem.product.name,
             description: cartItem.product.description,
             price: cartItem.product.price,
             quantity: cartItem.quantity,
-            image: { connect: { id: cartItem.product.image.id } },
+            image: { connect: { id: cartItem.product.photo.image.id } },
             user: { connect: { id: userId } },
           };
           return orderItem;
         });
-
+        console.dir(orderItems, { depth: null });
         // 5. create the Order
-        console.log('Creating the order');
         const order = await context.lists.Order.createOne({
           data: {
             total: charge.amount,
             charge: `${charge.id}`,
+            // TODO create array of items causes error
```

Comment:
@timleslie There create: orderItem (array of orderItem type) doesn't seem to work. 

Should I first create the orderItems, save their ids, and then connect them instead?